.eslintrc.json

```diff
@@ -0,0 +1,16 @@
+{
+    "env": {
+        "es6": true
+    },
+    "extends": "google",
+
+    "globals": {
+        "Atomics": "readonly",
+        "SharedArrayBuffer": "readonly"
+    },
+    "parserOptions": {
+        "ecmaVersion": 2018
+    },
+    "rules": {
+    }
+}
```

Comment:
Can you add a line  ?

Reply:
No problem. Sorry, I didn't notice the newline is missed.
Added.